mesonbuild/interpreter/interpreterobjects.py

```diff
@@ -309,6 +310,12 @@ def set_method(self, args: T.Tuple[str, T.List[str]], kwargs: 'EnvironmentSepara
         name, values = args
         self.held_object.set(name, values, kwargs['separator'])
 
+    @FeatureNew('environment.unset()', '1.3.0')
```

Comment:
None of the other `typed_pos_args` calls here have `()` in them, so I think we should be consistent and drop that

Reply:
I guess I copied this from somewhere else... Fixed.